dm/_utils/terror_gen/errors_release.txt

```diff
@@ -507,6 +507,7 @@ ErrSchedulerStarted,[code=46002:class=scheduler:scope=internal:level=medium], "M
 ErrSchedulerWorkerExist,[code=46003:class=scheduler:scope=internal:level=medium], "Message: dm-worker with name %s already exists"
 ErrSchedulerWorkerNotExist,[code=46004:class=scheduler:scope=internal:level=medium], "Message: dm-worker with name %s not exists"
 ErrSchedulerWorkerOnline,[code=46005:class=scheduler:scope=internal:level=medium], "Message: dm-worker with name %s is still online, Workaround: Please shut it down first."
+ErrSchedulerWorkerNotFree,[code=46033:class=scheduler:scope=internal:level=low], "Message: dm-worker with name %s not free"
```

Comment:
wrong error code order, you may need to move the ErrSchedulerWorkerNotFree in terror_list.go